### ISSUE TITLE
with_fx function

### ISSUE THREAD

**Issue by calcium3000** [gkvoelkl/python-sonic#4]
Good stuff!  Might I suggest adding in a with_fx() function?  Seems to be the only major omission, and seems easy enough to do.

**Comment by gkvoelkl** [comment]
Yes, fx is the next thing. But I think it is not so easy. I am waiting for the next version of sonic pi which will have communication between programs that I need for my implementation. What do you think?

**Comment by calcium3000** [comment]
Ah, yes I haven't actually tried it yet, but couldn't you just send it as part of the OSC message?  For example, for `play(60)` it looks like you'd be sending something like:
`/run-code use_synth :beep\nplay 60`

So couldn't you just add in a 'with_fx' parameter?  E.g.:
`/run-code use_synth :beep\nwith_fx :reverb\nplay 60`

**Comment by zakmcrbit** [comment]
great plugin! I would love to see With_FX too. Any plans / progress on this?

**Comment by gkvoelkl** [comment]
Hi I am working on it. Now there is a new sonic pi interface, perhaps that could work.
